action/stop.go

```diff
@@ -44,3 +45,11 @@ func (s stopAction) Execute(op types.OpType, filters []types.FilterType, items [
 
 	wg.Wait()
 }
+
+func getInstanceNames(instances []*types.Instance) []string {
+	result := make([]string, 0)
```

Comment:
We know the size, would be nice to init the array with that size

Reply:
ok, I'll fix it

Reply:
done